code/src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import './Header.css'
+
+export const Header = () => {
+
+    const logo = "https://www.themoviedb.org/assets/2/v4/logos/v2/blue_square_2-d537fb228cf3ded904ef09b136fe3fec72548ebc1fea3fbbd1ad9e36364db38b.svg";
+
+    return (
+        <header>            
+            {/* <div className="header-title-container">
```

Comment:
I've been told to remove code thats not being used. 
However personally I think it's interesting to see what people have been thinking about doing and then not used.

Reply:
@ckjerrgren Yeah I have a bit of a bad habit forgetting to remove these 🤣 I can agree with you that it's interesting to see what people have written before and not used 😜